t/05-scheduler-iso.t

```diff
@@ -64,4 +64,14 @@ is($schema->resultset("GruTasks")->search({taskname => 'download_asset'}), 0, 'g
 map { like($_, $expected, 'expected warning') } @warnings;
 is($schema->resultset("GruTasks")->search({taskname => 'download_asset'}), 1, 'gru task should be created');
 
+# Using non-asset _URL does not create gru job and schedule jobs
+output_like { $ids = OpenQA::Scheduler::Scheduler::job_schedule_iso(DISTRI => 'opensuse', VERSION => '13.1', FLAVOR => 'DVD', ARCH => 'i586', NO_ASSET_URL => 'nonexistent.iso') } qr/_URL downloading only allowed for asset types!/, $warning, 'expected debug info about non-asset type';
```

Comment:
Unfortunately this doesn't work anymore as now we do not use "Test::Output" anymore but "Test::Warnings" to be able to catch expected warnings as well as catch unexpected warnings. Test::Warnings has a different syntax and I think you did not catch this during rebase. travis complains about undefined variables e.g. "$ids". Probably you should write like in L63-L64, so your code should look something like:
```perl
my $warning = warning { my $ids = OpenQA::Scheduler::Scheduler::job_schedule_iso(DISTRI => 'opensuse', VERSION => '13.1', FLAVOR => 'DVD', ARCH => 'i586', NO_ASSET_URL => 'nonexistent.iso') };
$expected = qr/_URL downloading only allowed for asset types!/;
like($warning, $expected, 'expected debug info about non-asset type');
```
unless this is really output you want to check and not a "warning", then you might need to just reimport Test::Output again - also add it to cpanfile, then.